nuxt.config.js

```diff
@@ -239,6 +239,15 @@ module.exports = {
   axios: {
     // See https://github.com/nuxt-community/axios-module#options
   },
+  router: {
```

Comment:
So both the homepage and the newly created page by Ash (translation-test-page) require the same Vue page/component (index.vue). So instead of duplicating our code, force all routes which come after the forward-slash `/` ie (/translation-test-page) to be directed to the index.vue page. Sadly using `_.vue` wasn't sufficient enough to do what we wanted.

Reply:
using _.vue worked from a routing point of view. However on inside the _.vue file it wasn't aware of the current i18n locale.

Reply:
Understood.